packages/react-cosmos-playground/src/components/ResponsiveLoader/index.js

```diff
@@ -0,0 +1,183 @@
+
+    const { viewport = {} } = fixture;
+    const width =
+      viewport.width === 0 || viewport.width
+        ? viewport.width
+        : savedWidth || 320;
+    const height =
+      viewport.height === 0 || viewport.height
+        ? viewport.height
+        : savedHeight || 568;
+    const scale = true;
+
+    const { containerWidth, containerHeight } = this.state;
+    const scaleWidth = Math.min(1, containerWidth / width);
+    const scaleHeight = Math.min(1, containerHeight / height);
+    const scaleFactor = scale ? Math.min(scaleWidth, scaleHeight) : 1;
+    const scaledWidth = width * scaleFactor;
+    const scaledHeight = height * scaleFactor;
+
+    // We can't simply say
+    // if (!showResponsiveControls) { return <iframe> };
+    // because this causes flicker when switching between responsive and
+    // non responsive mode.
+
+    const containerClassName = showResponsiveControls
+      ? styles.container
+      : styles.nonResponsive;
+
+    const outerWrapperClassName = showResponsiveControls
+      ? styles.outerWrapper
+      : styles.nonResponsive;
+    const outerWrapperStyle = showResponsiveControls
+      ? { padding: PADDING }
+      : {};
+
+    const middleWrapperClassName = showResponsiveControls
+      ? ''
+      : styles.nonResponsive;
+    const middleWrapperStyle = showResponsiveControls
+      ? {
+          lineHeight: 0,
+          width: scaledWidth + 2 * BORDER_WIDTH,
+          height: scaledHeight + 2 * BORDER_WIDTH
+        }
+      : {};
+
+    const innerWrapperClassName = showResponsiveControls
+      ? classNames(styles.innerWrapper, styles.checkerboard)
+      : classNames(styles.nonResponsive, styles.checkerboard);
+    const innerWrapperStyle = showResponsiveControls
+      ? {
+          borderWidth: BORDER_WIDTH,
+          width: width + 2 * BORDER_WIDTH,
+          height: height + 2 * BORDER_WIDTH,
+          alignSelf: scaleHeight === 1 ? 'center' : 'flex-start',
+          justifySelf: scaleWidth === 1 ? 'center' : 'flex-start',
+          transform: `scale( ${scaleFactor} )`
+        }
+      : {};
+
+    return (
+      <div className={containerClassName}>
+        {showResponsiveControls && (
+          <Header
+            devices={devices}
+            dimensions={{ width, height, scale }}
+            containerWidth={containerWidth}
+            containerHeight={containerHeight}
+            updateDimensions={this.updateDimensions}
+          />
+        )}
+        <div
+          className={outerWrapperClassName}
+          ref={el => {
+            this.scalableDiv = el;
+          }}
+          style={outerWrapperStyle}
+        >
+          <div className={middleWrapperClassName} style={middleWrapperStyle}>
+            <div className={innerWrapperClassName} style={innerWrapperStyle}>
+              <iframe
+                ref={inputRef}
+                src={src}
+                frameBorder={0}
+                allowTransparency="true"
```

Comment:
![image](https://user-images.githubusercontent.com/250750/35796164-4a2e8748-0a64-11e8-80a5-33ab9af89f73.png)

Reply:
Seems like it does exist (but out of spec): https://stackoverflow.com/questions/3740700/there-is-no-attribute-allowtransparency

However we don't need it anyway, is just a legacy IE 8 thing. Deleted.